BotBrownCore/Workers/WorkerHost.cs

```diff
@@ -96,25 +92,8 @@ private void SpawnCommandWorker(CancellationToken cancellationToken)
                     return await commandWorker.Execute(cancellationToken);
                 }
             });
-        }
-
```

Comment:
We still need the webserver for the web api i think. Was this already deleted in this branch?